deploy/lingo_deploy.yaml

```diff
@@ -24,3 +24,14 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
```

Comment:
The healthz endpoint was setup already in main. This adds port/path to make use of it. Not sure about the time  values here